samples/packages/pulumi-typescript/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@openfabr/package-ri-pulumi",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "license": "AGPL-3.0-only",
+    "homepage": "https://openfabr.github.io/cdf",
+    "author": {
+        "name": "OpenFABR CDF Contributors",
+        "email": "open@fabrhq.com",
+        "url": "https://github.com/orgs/openfabr/teams/team-cdf"
+    },
+    "scripts": {
+        "test": "jest --silent=false"
+    },
+    "engines": {
+        "node": ">=16"
```

Comment:
Yep here it is, then you can lose `.nvmrc` file

Reply:
done